### ISSUE TITLE
[feature]: URL should be printed instead of URI in list of testcase at "/testlist/"

### ISSUE THREAD

**Issue by re-Tick** [keploy/ui#14]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

URL is not included in GraphQL query from backend. Please add that in your HttpReq GraphQL query and print URL in test case table instead of URI.

### Why should this be worked on?

Because user will be easily able to identify requests easily by URL instead of URI.

**Comment by nehagup** [comment]
Dependent on https://github.com/keploy/keploy/issues/44

**Comment by faraz16iqbal** [comment]
Hi @nehagup, is this issue open? Also can you please guide me as to how to proceed to fix this?

**Comment by nehagup** [comment]
@faraz16iqbal Yes it is, please take a look at the response of the testlist query.
